ts/screens/authentication/LandingScreen.tsx

```diff
@@ -77,25 +77,41 @@ const getCards = (
     id: 5,
     image: require("../../../img/landing/05.png"),
     title: I18n.t("authentication.landing.card5-title"),
-    content: I18n.t("authentication.landing.card5-content")
+    content: I18n.t("authentication.landing.card5-content"),
+    accessibilityLabel:
+      I18n.t("authentication.landing.accessibility.carousel.label") +
+      I18n.t("authentication.landing.card5-title") +
+      I18n.t("authentication.landing.card5-content-accessibility"),
```

Comment:
Should we consider to use template literal types to take care of "spaces" or "punctuation" between every single sentence?

```ts
accessibilityLabel: `${I18n.t(
  "authentication.landing.accessibility.carousel.label"
)} ${I18n.t("authentication.landing.card5-title")} ${I18n.t(
  "authentication.landing.card5-content-accessibility"
)}`,
```